Gemfile

```diff
@@ -183,6 +183,7 @@ group :test do
   gem 'rspec-sidekiq'
   gem 'rubocop-junit-formatter'
   gem 'rufus-scheduler'
+  gem 'rubyXL'
```

Comment:
You'll need to check-in the `Gemfile.lock` as well.

Reply:
I don't think rubyXL is used here? So this change in the gemfile should prob be reverted. It looks like `Roo` is used instead

Reply:
`rubyXL` is used to create the Excel file that is used in the unit spec, so I'll get it checked into the `Gemfile.lock`.

```ruby
    let(:uuid_list) do
      Tempfile.new(['uuid_list', '.xlsx']).tap do |file|
        workbook = RubyXL::Workbook.new
        ...
```